### PR TITLE
add support request best practices

### PR DESCRIPTION
**Pull request by wcpines**

### CODE REVIEW

File: data/topics.yml [hunk 1]
```diff
@@ -142,6 +142,8 @@
       url: "troubleshooting/app-processing-requests-slowly"
     - title: "The connection to my MySQL database is being refused, what should I do?"
       url: "troubleshooting/mysql-connection-refused"
+    - title: "What are support request best practices?"
+      url: "troubleshooting/support-ticket"
```
Comment (fancyremarker) [inline data/topics.yml]: Let's get the slug to be closer to the question, to really juice the most out of SEO.

```
url: troubleshooting/support-ticket-best-practices
```

Also, the more verbose title "What are the best practices when opening support tickets?" may be clearer.

File: source/topics/troubleshooting/support-ticket.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+When submitting a support request to Aptible, keep the following best practices in mind:
+
+1) If you provide us with as much detail as possible we'll be able to troubleshoot your issue much faster.  When appropriate, please provide
```
Comment (fancyremarker) [inline source/topics/troubleshooting/support-ticket.md]: These should be `1.` etc., to create ordered lists in Markdown.

File: source/topics/troubleshooting/support-ticket.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+When submitting a support request to Aptible, keep the following best practices in mind:
+
+1) If you provide us with as much detail as possible we'll be able to troubleshoot your issue much faster.  When appropriate, please provide  
+
+- Relevant handles (app, account, database etc)
```
Comment (fancyremarker) [inline source/topics/troubleshooting/support-ticket.md]: These bullets should be indented one or two spaces, to create a nested unordered list.

File: source/topics/troubleshooting/support-ticket.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+1) If you provide us with as much detail as possible we'll be able to troubleshoot your issue much faster.  When appropriate, please provide  
+
+- Relevant handles (app, account, database etc) 
+- Logs or error messages 
+- The date(s)/time(s) (UTC) when you experienced a problem
+
+2) Sanitize any sensitive information. This includes database connection URLs, SSH keys, passwords, tokens, and any secure environment variables you may be using. 
+
+3) We love getting formatted support requests because we're able to parse important information much easier.  Using backticks for monospacing or codeblocks is a great way to make information clear. You can either send us a ticket through the Zendesk webportal or include a [gist](https://gist.github.com/) in your request.
```
Comment (fancyremarker) [inline source/topics/troubleshooting/support-ticket.md]: Provide a link to the Zendesk web portal (also, let's go with "web portal").

**Comment by fancyremarker** [comment]
Content looks very good. All comments are nitpicks.

**Comment by gib** [comment]
Nice! Thanks @wcpines!

File: source/topics/troubleshooting/support-ticket.md (hunk 4 above)
Comment (wcpines) [inline source/topics/troubleshooting/support-ticket.md]: Looks like md actually isn't supported through the [portal](https://aptible.zendesk.com/hc/en-us/requests/new) either.  Maybe recommend gist?

Comment (fancyremarker) [inline source/topics/troubleshooting/support-ticket.md]: Sure. Let's specify "private gist." I would phrase it to be clear that this is not mandatory.

**Comment by fancyremarker** [comment]
Ping on this, @wcpines . Good to merge pending the changes above.

**Comment by wcpines** [comment]
Should be good to go.

**Comment by fancyremarker** [comment]
Great, merging.

**Pull request merged**